sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -115,23 +115,8 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit bc1a28ede0df7f837047b632e00437587672c134
-  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
-  # Date:   Mon Jul 29 16:44:58 2024 +0100
-  #     Merge pull request #1819 from DBDuncan/sean/rename-interop-to-external
-  #    [Bindless][Exp] Rename interop related structs/funcs with "external"
-  set(UNIFIED_RUNTIME_TAG bc1a28ede0df7f837047b632e00437587672c134)
-
-  set(UMF_BUILD_EXAMPLES OFF CACHE INTERNAL "EXAMPLES")
-  # Due to the use of dependentloadflag and no installer for UMF and hwloc we need
-  # to link statically on windows
-  if(WIN32)
-    set(UMF_BUILD_SHARED_LIBRARY OFF CACHE INTERNAL "Build UMF shared library")
-    set(UMF_LINK_HWLOC_STATICALLY ON CACHE INTERNAL "static HWLOC")
-  else()
-    set(UMF_DISABLE_HWLOC ${SYCL_UMF_DISABLE_HWLOC} CACHE INTERNAL "Disable hwloc for UMF")
-  endif()
```

Comment:
I think these changes are the ones making the CI failure, Is this related to the PR changes?

Reply:
gentle ping! @nrspruit as if this changes are not needed we could remove them and rerun CI.

Reply:
Fixed in my latest rebase.